frontend/vue/components/Syllabus/SyllabusForm.vue

```diff
@@ -147,7 +127,26 @@ export default defineComponent({
       this.lastDeletedCourse = undefined
     },
     submitForm () {
-      // TODO: add logic for submitting form
+      const url = this.editMode ? `/syllabus/${this.syllabus.id}` : '/syllabus'
+      const csrfToken = { _csrf: window.csrfToken }
+      fetch(url, {
+        method: 'POST', // *GET, POST, PUT, DELETE, etc.
+        headers: {'Content-Type': 'application/json' },
+        body: JSON.stringify({ 
+          ...csrfToken,
+          ...this.syllabus
+        })
+      }).then((res) => {
+        if (res.status === 200) {
+          res.json().then((jsonResult: Syllabus) => {
+            console.log(jsonResult)
+            window.location.href = `/syllabus/${jsonResult.code}`
+          })
+        } else {
+          // TODO: Manage this error (and improve backend feedback)
```

Comment:
comment for @gracelindsell — for future iterations, we could designs for use cases such as: _when a user tries to submit a syllabus, and something goes wrong... what do they see?_